packages/menu/src/MenuItem/MenuItem.tsx

```diff
@@ -156,6 +156,7 @@ export const MenuItem = InferredPolymorphic<MenuItemProps, 'button'>(
       </li>
     );
   },
+  'MenuItem',
 );
 
 MenuItem.displayName = 'MenuItem';
```

Comment:
is this line not relevant now?

Reply:
i just checked and i think its safe to remove for now